### PR TITLE
[videoplayer] DVDInputStream: Add an interface for stream selection.

### PR DESCRIPTION
**Pull request by ace20022**

### CODE REVIEW

File: xbmc/cores/VideoPlayer/DVDInputStreams/DVDInputStream.h [hunk 1]
```diff
@@ -146,6 +146,15 @@ class CDVDInputStream
     virtual void SetVideoResolution(int width, int height) {};
   };
 
+  class IStreams
```
Comment (FernetMenta) [inline xbmc/cores/VideoPlayer/DVDInputStreams/DVDInputStream.h]: that does not fit here. we deal with streams a demuxer level but not at this stage

Comment (ace20022) [reply xbmc/cores/VideoPlayer/DVDInputStreams/DVDInputStream.h]: But not for dvds and blurays. What if I move it to IMenus?

Comment (FernetMenta) [reply xbmc/cores/VideoPlayer/DVDInputStreams/DVDInputStream.h]: ok, try it.

**Comment by ace20022** [comment]
@FernetMenta better?

**Comment by FernetMenta** [comment]
@ace20022 yes, I think so. thanks!

**Comment by MartijnKaijser** [comment]
@ace20022 ready to go?
jenkins build this please

**Comment by ace20022** [comment]
I'm afraid, no. IIRC there are some stream number mapping issues. I'll reinvestigate it.

**Comment by phunkyfish** [comment]
PR closed as it appears to be abandoned by the author. 

If the author would like it kept open please just request so on the PR. If you feel there is still merit in merging this PR you can request the label be changed to `PR Cleanup: Backburner`.

**Pull request closed without merging**